test/integration_test.py

```diff
@@ -776,13 +776,26 @@ def test_azul_client_error_handling(self):
                           notifications)
 
 
-@unittest.skipIf(config.is_main_deployment(), 'Test would pollute portal DB')
-class PortalRegistrationIntegrationTest(IntegrationTestCase, AlwaysTearDownTestCase):
+class PortalTestCase(IntegrationTestCase):
 
     @cached_property
     def portal_service(self) -> PortalService:
         return PortalService()
 
+
+class PortalExpirationIntegrationTest(PortalTestCase):
+
+    def test_expiration_tagging(self):
+        s3_client = self.portal_service.client
+        response = s3_client.get_object_tagging(Bucket=self.portal_service.bucket,
```

Comment:
Does `self.portal_service.object_key` always exist in s3? What if it expired and was deleted? What would happen on a fresh deployment? `make deploy && make integration_test`

Reply:
Good catch